packages/p2p/src/components/buy-sell/rate-change-modal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Modal, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize, Localize } from 'Components/i18next';
+import './rate-change-modal.scss';
+
+const RateChangeModal = ({
+    local_currency,
+    should_show_rate_change_popup,
+    setShouldShowRateChangePopup,
+    setShouldShowBuySellForm,
```

Comment:
Why do we need to pass `should_shor_rate_change_popup`, `setShouldShowRateChangePopup` and `setShouldShowBuySellForm`, could we use the `buy_sell_store` using `useStores()` within the `RateChangeModal` instead?

Reply:
Yes we could. I am trying to follow consistency. The place where modal is used, the values and functions are passed as props

Reply:
Agree with @adrienne-deriv here. It makes sense to pass functions and values as props if a modal is used in multiple places with observables from different store affecting it's state. However, if this modal is used only in `buy-sell.jsx` component, we could just use the observables from `buy_sell_store` directly instead.